torch_em/data/datasets/README.md

```diff
@@ -0,0 +1,159 @@
+# Datasets in `torch-em`
```

Comment:
putting the markdown files into the python source code is not a good practice.

Let's create a separate folder `doc` for this. This can later be used for extra documentation files for building a documentation with pdoc.

Reply:
Done!